platform/linuxbsd/platform_gl.h

```diff
@@ -35,6 +35,10 @@
 #define GL_API_ENABLED // Allow using desktop GL.
 #endif
 
+#ifndef GLES_API_ENABLED
+#define GLES_API_ENABLED // Allow using GLES (ANGLE).
```

Comment:
It's not ANGLE but native GLES on Linux, no?

Reply:
Copy-pasted comment, it's not ANGLE.